plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1443,12 +1445,24 @@ local VocabBuilder = WidgetContainer:extend{
 
 function VocabBuilder:init()
     self.ui.menu:registerToMainMenu(self)
+    self:onDispatcherRegisterActions()
 end
 
 function VocabBuilder:addToMainMenu(menu_items)
     menu_items.vocabbuilder = {
         text = _("Vocabulary builder"),
         callback = function()
+            self:onShowVocabBuilder()
+        end
+    }
+end
+
+function VocabBuilder:prepareWidgetToShow()
+    if self.builder_widget then
```

Comment:
Why "builder_" in self.builder_widget ? You could just name it self.widget or self.voca_widget.
Oh, "builder" as in VocabularyBuilder :) I thought you might have picked this from our tools/wbuilder.lua :)
Anyway, "builder" in this context of building a widget :), a reader could think it's some kind of factory and get confused.

Reply:
Changed to `widget`. Right, the builder pattern, the wording can be somewhat misleading.